webhooks.md

```diff
@@ -2,6 +2,139 @@
 
 When an action of particular type is performed in the system, a message is sent to the configured webhook, using HTTP POST and JSON body.
 
+## General message
+
+```javascript
+{
+    "EnterpriseId": "851df8c8-90f2-4c4a-8e01-a4fc46b25178",
+    "IntegrationId": "c8bee838-7fb1-4f4e-8fac-ac87008b2f90",
+    "Events": [
+        {
+            "Discriminator": "ServiceOrderUpdated",
+            "Value": {
+                "Id": "bfee2c44-1f84-4326-a862-5289598f6e2d"
+            }
+        },
+        {
+            "Discriminator": "IntegrationCreated",
+            "Value": {
+                "Id": "c8bee838-7fb1-4f4e-8fac-ac87008b2f90",
+                "AccessToken":"9E5E84E9974D4F169662AB2200F27CB1-00B343A0DDA725CACAC028E38E3EABF",
+                "Requestor": null
+            }
+        }
+    ],
+    "Entities": {
+        "ServiceOrders": [
+            {
+                "AdultCount": 2,
```

Comment:
We should sort this logically according to how it is declared in the Dto documentation.